src/Npgsql/Internal/Resolvers/RangeTypeInfoResolver.cs

```diff
@@ -150,14 +144,14 @@ static void AddInfos(TypeInfoMappingCollection mappings, bool supportsMultiRange
             }
             else
             {
-                mappings.AddType<NpgsqlRange<DateTime>[]>(DataTypeNames.TsMultirange,
+                mappings.AddResolverType<NpgsqlRange<DateTime>[]>(DataTypeNames.TsMultirange,
```

Comment:
These were incorrect mappings, unnoticed as AddResolverType was aliased to AddType.